source/Plugins/Language/Swift/SwiftFormatters.cpp

```diff
@@ -1187,9 +1187,11 @@ bool lldb_private::formatters::swift::GLKit_SummaryProvider(
 
   Process &process = *process_sp.get();
 
-  // Get the type name without the "simd.simd_" prefix.
+  // Get the type name without the "GLKit." prefix.
```

Comment:
Isn't this behavior inconsistent with the new printing behavior now? We probably do want the prefix if we print it everywhere else?

Reply:
👍 This name is used for matching, not printing. The logic here is similar to what we were already doing for the (old) simd types above.

Reply:
Ah thanks!